src/NuGet.Core/NuGet.Protocol/Utility/OfflineFeedUtility.cs

```diff
@@ -208,6 +208,7 @@ public static void ThrowIfInvalid(string path)
                         var versionFolderPathResolver = new VersionFolderPathResolver(source);
 
                         using (var packageDownloader = new LocalPackageArchiveDownloader(
+                            string.Empty,
```

Comment:
Why not pass `source`?

If not `source`, `null` is a better choice than `string.Empty`.  Just use a named parameter or local constant to explain the literal.